.editorconfig

```diff
@@ -0,0 +1,19 @@
+# editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
More curious than anything but is there ever a reason to have this as false? I work with Python where not having this on is a compile error, so I turned it on 8 years ago and never looked back

Reply:
It's been taken from Unleash repo for consistency